proof/refine/AARCH64/CNodeInv_R.thy

```diff
@@ -5564,7 +5573,9 @@ lemma make_zombie_invs':
    apply (subgoal_tac "st_tcb_at' ((=) Inactive) p' s
                                \<and> obj_at' (Not \<circ> tcbQueued) p' s
                                \<and> bound_tcb_at' ((=) None) p' s
-                               \<and> ko_wp_at' (Not \<circ> hyp_live') p' s")
+                               \<and> ko_wp_at' (Not \<circ> hyp_live') p' s
+                               \<and> obj_at' (\<lambda>tcb. tcbSchedNext tcb = None
+                                                \<and> tcbSchedPrev tcb = None) p' s")
```

Comment:
probably too late, but did anyone consider an abbreviation for an unqueued tcb i.e. the `tcbSchedNext tcb = None ∧ tcbSchedPrev tcb = None` above?

Reply:
Corey wondered about an abbreviation for this, too, but it's only used with things related to finalising a TCB. I still think it's insignificant enough for it to not get its own abbreviation, but if I'm in the minority here I can definitely make an abbreviation for it. It's not quite exactly an unqueued TCB, for the reason I gave it another comment about the prev and next pointers and the flag.

Reply:
That's fine, after looking through the rest of the files, you're right that it doesn't pop up that often. Fine to keep as is.